<launch>
    <!-- Inicializar el nodo ROS -->
    <node name="grasp_server" pkg="action_server" type="ACT_SRVR_grasp.py" output="screen"/>
    <node name="pickup_action_server" pkg="action_server" type="ACT_SRVR_pickup.py" output="screen"/>
</launch>
